sentry/src/main/java/io/sentry/clientreport/ClientReportStorage.java

```diff
@@ -0,0 +1,9 @@
+package io.sentry.clientreport;
+
+import java.util.List;
+
+public interface ClientReportStorage {
```

Comment:
QuestionL Aren't interfaces always prefixed with `I` in the Java SDK?

Reply:
Not all, see e.g. `EventProcessor`

Reply:
@marandaneto do we have a preference?

Reply:
now with `I` prefix